README.md

```diff
@@ -205,6 +205,26 @@ Feel free to stop by our [Slack channel](https://empireslacking.herokuapp.com) (
 
 * The [SlithIR documentation](https://github.com/trailofbits/slither/wiki/SlithIR) describes the SlithIR intermediate representation.
 
+# FAQ
```

Comment:
We need to update the list of section at the top of the file to link to the FAQ section

Reply:
update table of contents